pycbc/events/significance.py

```diff
@@ -284,6 +349,28 @@ def check_significance_options(args, parser):
             parser.error("Threshold required for combo " + combo)
 
 
+def ifar_opt_to_far_limit(ifar_str):
+    """
+    Convert the string of an IFAR limit in years into a
+    float FAR limit in Hz.
+
+    Parameters
+    ----------
+    ifar_str: string
+        Upper limit on IFAR in years. Zero indicates no upper limit
+
+    """
+    ifar_float = positive_float(ifar_str)
+
+    try:
+        far_hz = 1. / (lal.YRJUL_SI * ifar_float)
+    except ZeroDivisionError:
```

Comment:
This error-as-corner-case switch feels bad to me .. what about 
`far_hz = 0. if (ifar_float == 0.) else 1. / (lal.YRJUL_SI * ifar_float)` ?

Reply:
That is probably the better way to do it